### PR TITLE
FF116 Relnote: Audio Output Devices API

### PR DESCRIPTION
**Pull request by hamishwillee**
FF116 enables the [Web Audio Output Devices API](https://developer.mozilla.org/en-US/docs/Web/API/Audio_Output_Devices_API) by default (except in Android) in https://bugzilla.mozilla.org/show_bug.cgi?id=1498512. This adds a release note and removes the experimental feature.

Other docs work can be tracked in #27748

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks Hamish, taking a look

File: files/en-us/mozilla/firefox/releases/116/index.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ This article provides information about the changes in Firefox 116 that affect d
 
 #### Media, WebRTC, and Web Audio
 
+- The [Audio Output Devices API](/en-US/docs/Web/API/Audio_Output_Devices_API) is now supported on all platforms except for Android.
+  This API allows web applications to redirect audio output to a permitted Bluetooth headset, speakerphone, or other device, instead of having to use the browser or underlying OS default.
+  Affected APIs include {{domxref("MediaDevices.selectAudioOutput()")}}, {{domxref("MediaDevices.enumerateDevices()")}}, [`HTMLMediaElement.setSinkId()`](/en-US/docs/Web/API/HTMLMediaElement/setSinkId), [`HTMLMediaElement.sinkId`](/en-US/docs/Web/API/HTMLMediaElement/sinkId), and the permission policy [`Permissions-Policy: speaker-selection`](/en-US/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection).
+  ([Firefox bug 1498512](https://bugzil.la/1498512)).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/116/index.md:48]: Formatting nit:

```suggestion
  Affected APIs include {{domxref("MediaDevices.selectAudioOutput()")}}, {{domxref("MediaDevices.enumerateDevices()")}}, [`HTMLMediaElement.setSinkId()`](/en-US/docs/Web/API/HTMLMediaElement/setSinkId), [`HTMLMediaElement.sinkId`](/en-US/docs/Web/API/HTMLMediaElement/sinkId), and the permission policy [`Permissions-Policy: speaker-selection`](/en-US/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection ([Firefox bug 1498512](https://bugzil.la/1498512)).
```

**Review by bsmth — APPROVED** [review]
Looks great with a formatting suggestion.

Thank you!

**Comment by hamishwillee** [comment]
Thanks @bsmth !

**Pull request merged**